tailscale/rootfs/etc/s6-overlay/s6-rc.d/proxy/run

```diff
@@ -31,6 +31,13 @@ if bashio::var.true "$(bashio::core.ssl)"; then
   bashio::exit.nok
 fi
 
+# Test Home Assistant's HTTP reverse proxy configuration
+if (( 200 != $(curl -s -o /dev/null -w "%{http_code}" "http://127.0.0.1:$(bashio::core.port)" -H "X-Forwarded-For: 127.0.0.1") )); then
+  bashio::log.error "Unable to connect to Home Assistant on http://127.0.0.1:$(bashio::core.port)"
```

Comment:
Let's remove the technical detail here, it can be confusing as lots of users won't care/know what `127.0.0.1` is doing here.

```suggestion
  bashio::log.error "Unable to connect to Home Assistant."
```